lib/middleware/websocket/client.rb

```diff
@@ -1,10 +1,13 @@
 module Websocket
   class Client
-    attr_reader :connection_client, :client_id
+    attr_reader :connection_client, :client_attributes
+    attr_accessor :position
 
     def initialize(connection_client: client, client_id: client_id)
       @connection_client = connection_client
       @client_id = client_id
+      @position = '0%'
+      @client_attributes = { 'id' => @client_id, 'position' => @position }
```

Comment:
I would pull `client_attributes` into a method to ensure that we get up to date information each time we call it. Right now we'll only ever get the initial values, which won't mean a lot once the game starts.

Reply:
@jgayfer implemented.